packages/abi-utils/lib/arbitrary.ts

```diff
@@ -393,29 +392,46 @@ const reservedWords = new Set([
   "years"
 ]);
 
-// borrowed from https://runkit.com/dubzzz/faker-to-fast-check
-const fakerToArb = (template: string, transform = camelCase) => {
+type WordListKey = keyof typeof wordLists;
+const getArb = (
+  wordTypes: WordListKey[],
+  transform = camelCase
+): fc.Arbitrary<string> => {
+  const results: fc.Arbitrary<string>[] = [];
+  for (const wordType of wordTypes) {
```

Comment:
This seems overly complicated to me in two ways:

1. Rather than having these separate noun-vs-verb branches, you could just use `wordLists[wordType]` everywhere with no branch.
2. Instead of doing `fc.integer(...).noBias().noShrink().map(...)`, you could just use `fc.constantFrom(...)`, if I'm understanding correctly?

It looks to me like doing those would simplify this function a lot.

Reply:
> 2\. Instead of doing `fc.integer(...).noBias().noShrink().map(...)`, you could just use `fc.constantFrom(...)`, if I'm understanding correctly?

OMG I was looking for this very thing!

@eggplantzzz I'll address this concern and push up a new commit, since this is my mess that Harry's showing issue with.

Reply:
Updated. Wow that's so much cleaner. Thanks @haltman-at!